app/javascript/components/admin/server-recordings/ServerRecordingRow.jsx

```diff
@@ -0,0 +1,49 @@
+import { TrashIcon } from '@heroicons/react/outline';
+import React from 'react';
+import PropTypes from 'prop-types';
+import RecordingRow from '../../shared/RecordingRow';
+import Modal from '../../shared/Modal';
+import DeleteRecordingForm from '../../forms/DeleteRecordingForm';
+import useUpdateRecordingVisibility from '../../../hooks/mutations/recordings/useUpdateRecordingVisibility';
+import useUpdateRecording from '../../../hooks/mutations/recordings/useUpdateRecording';
+import useDeleteRecording from '../../../hooks/mutations/recordings/useDeleteRecording';
+
+export default function ServerRecordingRow({ recording }) {
+  // TODO: Change all mutations when their APIs becomes ready.
+  return (
+    <RecordingRow
+      recording={recording}
+      visibilityMutation={useUpdateRecordingVisibility}
+      updateMutation={useUpdateRecording}
```

Comment:
@hadicheaito1 for #3258 inject your mutation in lace of `useUpdateRecording`.

Reply:
Perfect, thanks Amir!

Reply:
You're welcome ^^